diff_cover/violationsreporters/violations_reporter.py

```diff
@@ -336,22 +338,62 @@ def parse(lcov_file):
                 if line_no not in lcov_report[source_file]:
                     lcov_report[source_file][line_no] = 0
                 lcov_report[source_file][line_no] += num_executions
+            elif directive == "BRDA":
+                args = content.split(",")
+                if len(args) != 4:
+                    continue
+                line_no = int(args[0])
```

Comment:
Again cue from above
```suggestion
                if source_file is None:
                    raise ValueError(
                        f"No source file specified for line coverage: {line}"
                    )
                line_no = int(args[0])
```